pkg/platform/kube/ingress/ingress.go

```diff
@@ -488,6 +492,45 @@ func (m *Manager) compileBasicAuthAnnotationsAndSecret(ctx context.Context, spec
 	return ingressAnnotations, secret, nil
 }
 
+func (m *Manager) compileSSOAuthAnnotations(spec Spec) (map[string]string, error) {
+	var signinURL, authURL string
+	// Use auth and signin URLs from the spec if provided; otherwise, fallback to platform configurations
```

Comment:
As discussed, the auth and sign-in urls should only be taken from the platform configuration, and not from the api gateway spec.